pkg/config/config.go

```diff
@@ -677,6 +680,14 @@ func IsKubernetes() bool {
 	return false
 }
 
+// IsDockerSwarm returns whether the Agent is running on a Swarm cluster
+func IsDockerSwarm() bool {
+	if os.Getenv("STS_DOCKER_SWARM") != "" {
```

Comment:
Why are we prefixing this with `STS_`?

Reply:
I did this because all environment introduced by us is prefixed with `STS`. so Tiziano told me to do it for this as well.